web/templates/layout/app.html.eex

```diff
@@ -24,6 +24,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title><%= title_or_default(assigns[:title]) %></title>
+    <%= if description = assigns[:description] do %>
+    <meta name="description" content="<%= description %>">
```

Comment:
Don't need to fix it, but in the future please indent the content of `if` blocks.